### ISSUE TITLE
Does not work with translucent status bar Activities.

### ISSUE THREAD

**Issue by iamBedant** [saket/FluidKeyboardResize#12]
If you add `<item name="android:windowTranslucentStatus">true</item>` to your activity theme, the animation does not work. Reason is FrameLayout(android.R.id.content) does not get resized in case of such activities.
<img width="1680" alt="Screenshot 2019-10-15 at 12 35 29 PM" src="https://user-images.githubusercontent.com/6387538/66853575-3d5e3980-ef9d-11e9-9a0a-5a298fdbc6ca.png">

**Comment by saket** [comment]
Were you able to check if any other ViewGroup gets resized? Afaik fullscreen activities aren't resized, but this sounds different

**Comment by iamBedant** [comment]
Nope. none of the viewgroup gets resized.

**Comment by saket** [comment]
Does setting `android:fitsSystemWindows="true"` on the root ViewGroup in the layout change anything?

**Comment by iamBedant** [comment]
Nope. The above screenshot is with `android:fitsSystemWindows="true"` only. Without it, the EditText will not even move up. It will be covered by the soft keyboard.

**Comment by saket** [comment]
Ah okay. I'm afraid I don't have any answers for this ¯\\_(ツ)_/¯

**Comment by fic212** [comment]
Hi. It seems that, when using "android:windowTranslucentStatus" in theme, android does not resize any views, but adds bottom padding to a root view of your activity (not content view, but root view). Don't know why, it doesn't seems like right thing to do, but they did it that way.

I still manage to detect keyboard change and animate it by animating bottom margin to the opposite direction of bottom padding, but it seams to hacky so I would not recommend it. 

EDIT: Actually, bottom padding not necessary applied on root view, but on a view that has fitsSystemWindows properties set on true.

**Comment by saket** [comment]
Yea this whole implementation was a giant hack anyway. Hopefully we won't have to do any of this once [`WindowInsetsController`](https://developer.android.com/reference/android/view/WindowInsetsController) gets backported as part of androidx.

**Comment by kkgosu** [comment]
I've faced problem when resizing don't work with _decorView.systemUiVisibility = View.SYSTEM_UI_FLAG_LAYOUT_FULLSCREEN_. So this article helped me to solve the problem https://medium.com/@sandeeptengale/problem-solved-3-android-full-screen-view-translucent-scrollview-adjustresize-keyboard-b0547c7ced32

[My commit with fix ](https://github.com/kkgosu/Emojify/commit/be6a4472bf838c0a2d329a65bbc4b50ec7f6664a)

**Comment by po1xiao** [comment]
I got an idea form [drakeet](https://github.com/drakeet), you can see if it helps.

set the target activity's windowSoftInputMode as adjustNothing, then add an invisible window on the top of the activty, and set the window's windowSoftInputMode as adjustResize, when the keyboard visibility changes the window size will change too, so we can know the keyboard show or hide according to the window's size change, then wo can tell activity to do the size changing animation.
